googleapiclient/discovery.py

```diff
@@ -921,13 +925,18 @@ def method(self, **kwargs):
   return (methodName, method)
 
 
-def createNextMethod(methodName):
+def createNextMethod(methodName, pageToken, nextPageToken,
+                     isPageTokenParameter):
   """Creates any _next methods for attaching to a Resource.
 
   The _next methods allow for easy iteration through list() responses.
 
   Args:
     methodName: string, name of the method to use.
+    pageToken: string, name of request page token field.
+    nextPageToken: string, name of response page token field.
```

Comment:
Would `nextPageTokenField` or `nextPageTokenFieldName` be a better fit?

(Also I really hate this lowerCamelCase nonsense, but I know you're just fitting in with existing style)

Should this have a default value?